directory/tests/test_directory.py

```diff
@@ -288,6 +288,30 @@ def test_file_metrics_many(aggregator):
         assert aggregator.metrics_asserted_pct == 100.0
 
 
+def test_folder_metrics(aggregator):
+    config_stubs = common.get_config_stubs(temp_dir + "/main", collect_folder_stats=True)
+
+    for config in config_stubs:
+        aggregator.reset()
```

Comment:
Can we `pytest.parametrize` over `config_stubs`? If yes, we can remove this line, since the `aggregator` fixture gets reset between each test.